content/en/docs/tasks/administer-cluster/static-pod.md

```diff
@@ -89,8 +89,13 @@ static-web-my-node1        1/1       Running   0          2m
 
 Labels from the static pod are propagated into the mirror-pod and can be used as usual for filtering.
 
-Note that we should make sure kubelet have enough permissions to create the mirror pod in API server, otherwise the creation request may be forbidden by API server (e.g. [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)).
-Also we cannot delete the pod with the API server (e.g. via [`kubectl`](/docs/user-guide/kubectl/) command), kubelet simply won't remove it.
+{{< note >}}
+**Note**: Make sure the Kubelet have permission to create the mirror pod in the API server.
+If not, the creation request is forbidden by the API server. See
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/). A permission error also
+causes the API server to fail to delete the pod. If you try, such as by using kubectl, the
+Kubelet cannot remove it.
```

Comment:
Sorry, I should split these two sentences.
In fact the `delete` description has nothing to do with permissions.
It means even if you delete the mirror pod with `kubectl delete`, the kubelet will create it again.

Reply:
> If not, the creation request is forbidden by the API server.

This sentence can be removed. Or you can change "forbidden" to "rejected".

> If you try, such as by using kubectl, the Kubelet cannot remove it.

This sentence can be removed.

Reply:
@tengqm 
updated.